static/gsApp/views/seerAutomation/trial.tsx

```diff
@@ -0,0 +1,165 @@
+      return;
+    }
+
+    // If you've already got Seer, then go to settings and you should see the new ones.
+    if (organization.features.includes('seat-based-seer-enabled')) {
+      navigate(normalizeUrl(`/organizations/${organization.slug}/settings/seer/`));
+      return;
+    }
+  }, [navigate, organization.features, organization.slug]);
+
+  return (
+    <Fragment>
+      <Flex justify="center">
+        <img
+          src={seerConfigMain}
+          alt="Seer"
+          style={{
+            maxWidth: '90%',
+            marginTop: '-40px',
+            marginBottom: '-77px',
+            transform: 'rotate(1.281deg)',
+          }}
+        />
+      </Flex>
+
+      <Container
+        padding="2xl xl"
+        border="primary"
+        radius="lg"
+        background="secondary"
+        maxWidth="600px"
+        margin="auto"
+      >
+        <Stack gap="lg">
+          <Heading as="h1" align="center">
+            {t('Say Hello to a Smarter Sentry')}
+          </Heading>
+          <Text>
+            {tct(
+              `Meet Seer: Sentry’s AI agent that helps you troubleshoot and fix
+            problems with your application, review your PRs, propose solutions
+            to your bugs and performance issues, and even partner with coding
+            agents to implement fixes in code. Learn more about Seer [link:here]!`,
+              {
+                link: (
+                  <ExternalLink href="https://docs.sentry.io/product/ai-in-sentry/seer/" />
+                ),
+              }
+            )}
+          </Text>
+
+          <Heading as="h3" align="center">
+            {t('Seer is able to...')}
+          </Heading>
+          <Text as="div" bold>
+            <Grid
+              as="ul"
+              columns="repeat(2, 1fr)"
+              gap="xl 2xl"
+              padding="lg"
+              style={{listStyle: 'none'}}
+            >
+              {BUTTONS.map(({label, imageSrc, href}) => {
+                return (
+                  <Container
+                    key={`seer-automation-trial-${label}`}
+                    as="li"
+                    border="primary"
+                    radius="md"
+                    background="tertiary"
+                    padding="md"
+                    style={{position: 'relative'}}
+                  >
+                    <InteractionStateLayer />
+                    <ExternalLink href={href}>
+                      <Flex gap="lg" align="center">
+                        <img width="44" height="44" alt="" src={imageSrc} />
+                        {label}
+                      </Flex>
+                    </ExternalLink>
+                  </Container>
+                );
+              })}
+            </Grid>
+          </Text>
+          <Flex align="center" justify="center" paddingTop="lg">
+            {hasBillingAccess(organization) ? (
+              <LinkButton
+                href="/settings/billing/overview/?product=seer"
```

Comment:
Should this be `to`?

Reply:
d'oh. that's why the bots are flagging it.